docs/layers/grid-layer.md

```diff
@@ -15,28 +15,24 @@ Inherits from all [Base Layer](/docs/layers/base-layer.md) properties.
 
 ## Layer-specific Properties
 
-##### `latDelta` (Number, optional)
+##### `latOffset` (Number, optional)
```

Comment:
I think we need to drop the lat/lng here. But we didn't conclude on it in the review.